src/js/components/PodContainerSpecView.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+
+import DescriptionList from './DescriptionList';
+import EnvironmentList from './EnvironmentList';
+import ServiceConfigUtil from '../utils/ServiceConfigUtil';
+
+class PodContainerSpecView extends React.Component {
+  getArtifactsSection() {
+    let {container: {artifacts=[]}} = this.props;
+
+    if (artifacts.length === 0) {
+      return null;
+    }
+
+    let keysMapping = {
+      uri: 'URI',
+      extract: 'Extract',
+      executable: 'Executable',
+      cache: 'Cache',
+      destPath: 'Destination Path'
+    };
+
+    let nodes = artifacts.map(function (artifact, i) {
+      return (
+        <DescriptionList
+          key={i}
+          hash={artifact}
+          renderKeys={keysMapping} />
+      );
+    });
+
+    return (
+      <div>
+        <h5 className="inverse flush-top">
+          Artifacts
+        </h5>
+        {nodes}
+      </div>
+    );
+  }
+
+  getGeneralDetails() {
+    let hash = Object.assign({}, this.props.container);
+
+    // Handled separately
+    delete hash.endpoints;
+    delete hash.artifacts;
+    delete hash.volumeMounts;
+    delete hash.environment;
```

Comment:
You could use `Util#omit` and that way you don't need to clone the object here and it's a bit cleaner.